build-tools/mono-runtimes/mono-runtimes.targets

```diff
@@ -4,12 +4,26 @@
   <PropertyGroup>
     <_SourceTopDir>..\..</_SourceTopDir>
     <_BclFrameworkDir>$(OutputPath)\lib\xbuild-frameworks\MonoAndroid\v1.0</_BclFrameworkDir>
+    <_MandroidInternalDir>$(OutputPath)\lib\mandroid-internal</_MandroidInternalDir>
```

Comment:
This shouldn't be `lib\mandroid-internal`, just `lib\mandroid`. `mdoc.exe` is installed on macOS as `/Library/Frameworks/Xamarin.Android.framework/Libraries/mandroid/mdoc.exe`.